pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.props

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <BuildRID Include="alpine.3.4.3-x64" />
+    <BuildRID Include="debian.8-x64" />
+    <BuildRID Include="fedora.23-x64" />
+    <BuildRID Include="fedora.24-x64" />
+    <BuildRID Include="opensuse.13.2-x64" />
+    <BuildRID Include="opensuse.42.1-x64" />
+    <BuildRID Include="osx.10.10-x64">
+      <OSGroup>OSX</OSGroup>
+    </BuildRID>
+    <BuildRID Include="rhel.7-x64" />
+    <BuildRID Include="ubuntu.14.04-x64" />
+    <BuildRID Include="ubuntu.16.04-x64" />
+    <BuildRID Include="ubuntu.16.10-x64" />
+    <BuildRID Include="win7-x86">
+      <Platform>x86</Platform>
+      <OSGroup>Windows_NT</OSGroup>
+    </BuildRID>
+    <BuildRID Include="win7-x64">
+      <OSGroup>Windows_NT</OSGroup>
+    </BuildRID>
+    <BuildRID Include="win10-x86">
+      <Platform>x86</Platform>
+      <OSGroup>Windows_NT</OSGroup>
+    </BuildRID>
+    <BuildRID Include="win10-x64">
+      <OSGroup>Windows_NT</OSGroup>
+    </BuildRID>
+    <!-- Omit until we have official build support from CoreFx
```

Comment:
We already build packages for win-arm64 and win8-arm today. Is there a specific reason to not enable them right now as it will breaks folks (like IoT and PS) who consume the SharedFW from the feed on a periodic basis?